Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/log.go

```diff
@@ -201,5 +205,22 @@ func (o LogsOptions) RunLog() (int64, error) {
 	}
 	defer readCloser.Close()
 
+	if file, ok := o.Out.(*os.File); ok {
+		inFd := file.Fd()
+		if term.IsTerminal(inFd) {
+			defer fmt.Fprint(o.Out, "\033[0m")
+			ch := make(chan os.Signal, 1)
+			signal.Notify(ch, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT)
+			defer signal.Stop(ch)
+			go func() {
+				if _, ok := <-ch; !ok {
+					return
+				}
```

Comment:
It's probably cleaner to wrap out with something that interprets close and writes the clear, if a terminal is in use.  However, what commands do this today?  Does ssh clear the terminal state?  What about tail?